src/main/java/com/example/onlineshop_frontend/clients/AdminClient.java

```diff
@@ -0,0 +1,16 @@
+package com.example.onlineshop_frontend.clients;
+
+import com.example.onlineshop_frontend.dto.EmployeeDTO;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@FeignClient(name = "onlineshop-api-admin", url = "http://localhost:8082/admin")
```

Comment:
Let's move the url bit in the application.properties file - onlineshop.api.url=http://localhost:8082
and then this will change to: @FeignClient(name = "onlineshop-api", url = "${onlineshop.api.url}")

All methods will need to be changed to append /admin like so:
instead of: @GetMapping("/show-employees")
we'll have: @GetMapping("/admin/show-employees")

That way we'll have a centralized place for the backend URL and we'll be more flexible